tests/suites/deploy/deploy_default_series.sh

```diff
@@ -28,14 +41,27 @@ run_deploy_not_default_series() {
 	ensure "${model_name}" "${file}"
 
 	juju model-config default-series=bionic
-	juju deploy ubuntu --series focal
-	juju deploy cs:ubuntu csubuntu --series focal
+	juju deploy ubuntu --series focal --storage "files=tmpfs"
+	wait_for "ubuntu" "$(idle_condition "ubuntu")"
+	series=$(juju status --format=json | jq ".applications.ubuntu.series")
+	echo "$series" | check "focal"
 
-	ubuntu_series=$(juju status --format=json | jq ".applications.ubuntu.series")
-	echo "$ubuntu_series" | check "focal"
+	destroy_model "${model_name}"
+}
 
-	csubuntu_series=$(juju status --format=json | jq ".applications.csubuntu.series")
-	echo "$csubuntu_series" | check "focal"
+run_deploy_not_default_series_cs() {
+	echo
+
+	model_name="test-deploy-not-default-series-cs"
+	file="${TEST_DIR}/${model_name}.log"
+
+	ensure "${model_name}" "${file}"
+
+	juju model-config default-series=bionic
+	juju deploy cs:ubuntu --series focal
```

Comment:
if the above question is answered and we need storage, should this not also have the `--storage` directive?

Reply:
thanks, missed one